examples/mbedtls/Makefile

```diff
@@ -0,0 +1,34 @@
+############################################################################
+# apps/examples/mbedtls/Make.defs
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.  The
+# ASF licenses this file to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance with the
+# License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+############################################################################
+
+include $(APPDIR)/Make.defs
+
+# mbedtls built-in application info
+
+PROGNAME  = $(CONFIG_EXAMPLES_MBEDTLS_PROGNAME)
+PRIORITY  = $(CONFIG_EXAMPLES_MBEDTLS_PRIORITY)
+STACKSIZE = $(CONFIG_EXAMPLES_MBEDTLS_STACKSIZE)
+MODULE    = $(CONFIG_EXAMPLES_MBEDTLS)
+
+# Mbed TLS Demo
+
+MAINSRC = mbedtls_main.c
```

Comment:
For an example, an alternative is to use the one of those comming with mbedTLS, I've got this:

`MAINSRC = $(APPDIR)/external/mbedtls/mbedtls/programs/ssl/ssl_server.c`

Reply:
Yep just using this as a minimal test for now.